src/strategies/math/examples.json

```diff
@@ -1,4 +1,106 @@
+    "strategy": {
+      "name": "math",
+      "params": {
+        "symbol": "MATH",
+        "operands": [
+          {
+            "type": "strategy",
+            "strategy": {
+              "network": "1",
+              "name": "contract-call",
+              "params": {
+                "args": ["%{address}"],
+                "symbol": "Locked CVX",
+                "address": "0x96C68D861aDa016Ed98c30C810879F9df7c64154",
+                "decimals": 18,
+                "methodABI": {
+                  "name": "getUserInfo",
+                  "type": "function",
+                  "inputs": [
+                    {
+                      "name": "_account",
+                      "type": "address",
+                      "internalType": "address"
+                    }
+                  ],
+                  "outputs": [
+                    {
+                      "name": "totalDeposited",
+                      "type": "uint256",
+                      "internalType": "uint256"
+                    }
+                  ],
+                  "constant": true,
+                  "stateMutability": "view"
+                }
+              }
+            }
+          },
+          {
+            "type": "strategy",
+            "strategy": {
+              "network": "1",
+              "name": "contract-call",
+              "params": {
+                "args": ["%{address}"],
+                "symbol": "Locked CVX",
+                "address": "0x96C68D861aDa016Ed98c30C810879F9df7c64154",
+                "decimals": 18,
+                "output": "totalBorrowed",
+                "methodABI": {
+                  "name": "getUserInfo",
+                  "type": "function",
+                  "inputs": [
+                    {
+                      "name": "_account",
+                      "type": "address",
+                      "internalType": "address"
+                    }
+                  ],
+                  "outputs": [
+                    {
+                      "name": "totalDeposited",
+                      "type": "uint256",
+                      "internalType": "uint256"
+                    },
+                    {
+                      "name": "totalPendingUnlocked",
+                      "type": "uint256",
+                      "internalType": "uint256"
+                    },
+                    {
+                      "name": "totalUnlocked",
+                      "type": "uint256",
+                      "internalType": "uint256"
+                    },
+                    {
+                      "name": "totalBorrowed",
+                      "type": "uint256",
+                      "internalType": "uint256"
+                    }
+                  ],
+
+                  "constant": true,
+                  "stateMutability": "view"
+                }
+              }
+            }
+          }
+        ],
+        "operation": "minus"
+      }
+    },
```

Comment:
Can add smaller examples plz 😄 maybe like ticket strategy or something, Everything else looks fine, thanks for the updates